core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/internal/ResourceDocument.java

```diff
@@ -62,8 +62,8 @@
 )
 @JsonDeserialize(builder = ResourceDocument.Builder.class)
 @Script(
-		name="typeSort", 
-		script="return params.orderByType.getOrDefault(doc.resourceType.value, doc.resourceType.value)")
+		name="typeRank", 
+		script="return params.ranks.getOrDefault(doc.resourceType.value, Integer.MAX_VALUE)")
```

Comment:
Ranks are single-digit `String` literals, while the default value is an `Integer` now. One or the other type should be chosen 😃

Reply:
Right, I forgot to mention the change of getRank method's return type to int. 😭

Reply:
Done

Reply:
Done